website/docs/components/segmented-group/partials/code/component-api.md

```diff
@@ -0,0 +1,29 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Contextual components
+
+The following predefined segments can be passed into the Segmented Group as yielded components: `Button`, `Dropdown`, `Select`, `TextInput`. For bespoke segments use the `Generic` block and style it accordingly.
```

Comment:
I've been referring to segments as a component, "Segments", mainly because they must be defined as components in Figma. Should this be the case everywhere?

Reply:
agree – will update